docs/decisions/0002-content-flexibility.rst

```diff
@@ -0,0 +1,52 @@
+2. Approach to Content Flexibility
+==================================
+
+Context
+-------
+
+Open edX courses follow a strict Course > Section > Subsection > Unit > Module hierarchy. There are a number of use cases that do not fit this pattern:
+
+* A problem bank shared via LTI, with individual problems for use in other LMS systems.
+* Short courses that do not require the "Section" middle layer of hierarchy.
+* Standalone assessments given after an informational session.
+* LabXchange-style pathways and clusters.
+
+The Learning Core should have a set of composable primitives that could be configured to construct these sorts of experiences, while still maintaining the ability to support the traditional Open edX course format.
+
+Decision
+--------
+
+The following are foundational, extensible concepts in the Learning Core, which can be combined in different ways:
+
+Block
```

Comment:
Maybe this should be "Item" instead, to differentiate it more from structural XBlocks, and to be more in line with QTI terminology?

Reply:
> One thing I realized about Atom is that if we have a project about Atomic Learning Units and adopt that language, it's going to be painful because Atoms in the data model would actually be one or two layers more granular than what we're presenting to the user.

@ormsbee You're right, that'd be an unfortunate mismatch in terminology. We should call the blocks `Hadrons` (just kidding :).

This has me questioning the name of the project now! "Atom" comes from the Greek word "indivisible". The name "Atomic Learning Units" implies a system of pieces learning content that cannot be broken down further, but doesn't speak the intended outcome of the project (which is to have "Composable" or "Modular" learning units, right?).

Reply:
Yeah that's fair!